molgenis-core-ui/src/main/javascript/molgenis-global-webpack.js

```diff
@@ -1,25 +1,19 @@
-'use strict';
 
-//######################################################################
-//##### Anything required here will get bundled                    #####
-//##### into molgenis-bundle.js                                    #####
-//######################################################################
+//##############################################################################
+//##### Anything required here will get bundled                            #####
+//##### into molgenis-core-ui.bundle.js                                    #####
+//##############################################################################
 
-var $ = window.$ = window.jQuery = require('jquery');
-var Bootstrap = require('bootstrap');
-var React = require('react');
-var _ = require('underscore');
-var validate = require('jquery-validation');
-var Handlebars = require('handlebars/dist/handlebars.min');
 window.molgenis = require('./modules/MolgenisGlobalObject');
+window.$ = require('jquery');
```

Comment:
If you define this here AND in the usermanager.js, won't webpack complain about double declaration?